OpenRA.Mods.Common/Widgets/Logic/Lobby/LobbyLogic.cs

```diff
@@ -25,6 +25,7 @@ namespace OpenRA.Mods.Common.Widgets.Logic
 {
 	public class LobbyLogic : ChromeLogic
 	{
+		public const string OpenRALANGame = "OpenRALANGame";
```

Comment:
`LobbyLogic` runs client side for every player who joins a server, not on the server itself.  This distinction is important for dedicated servers.

Yes, please do advertise started games.